sc_preprocessing/2_make_pseudobulks/scanpy_sc_preprocess_allSplits-liver.ipynb

```diff
@@ -0,0 +1,1899 @@
+{
```

Comment:
<p>as a general comment: in the other notebook you had a really nice description of what is sample_id, stim, etc. i think it could help follow this NB if you add it here also.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/buddi_analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>good point, added in: <a href="https://github.com/greenelab/buddi_analysis/pull/2/commits/9717583b42fad0318684ccf25adb4513954ca3d3" target="_blank">https://github.com/greenelab/buddi_analysis/pull/2/commits/9717583b42fad0318684ccf25adb4513954ca3d3</a></p>